homework/index.js

```diff
@@ -93,18 +116,21 @@
         cellContent.textContent = repo[content[i].attribute];
       }
     }
-    fetchJSON(repo.contributors_url, (error, contributors) => renderContributors(contributors));
+
+    fetchJSON(repo.contributors_url)
+      .then(contributors => renderContributors(contributors))
+      .catch(error => {
+        console.error(error);
+      });
   }
 
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
+    fetchJSON(url)
+      .then(data => {
         let optionArr = data;
```

Comment:
`optionsArr`? I think you get an array of repository objects, so a name such as `repositories` would make more sense to me. And I would use that name direct;y as the parameter name of the `.then()` method, so you do not have to use line 130:

```js
.then( repositories => {
```

Reply:
It is done